src/Router.php

```diff
@@ -178,21 +178,10 @@ public static function is_graphql_http_request() {
 					return false;
 				}
 
-				$full_path = $host . $uri;
-				$site_url  = site_url( self::$route );
-
-				// Strip protocol.
-				$replaced_path = preg_replace( '#^(http(s)?://)#', '', $full_path );
-				if ( ! empty( $replaced_path ) ) {
-					$full_path = $replaced_path;
-				}
-				$replaced_url = preg_replace( '#^(http(s)?://)#', '', $site_url );
-				if ( ! empty( $replaced_url ) ) {
-					$site_url = $replaced_url;
-				}
+				$len                     = strlen( self::$route );
```

Comment:
Thinking this might be a better one. and eliminates routes like `/graphql/test` triggering as valide. not sure if there's a performance difference

```suggestion
								$is_graphql_http_request = boolval( preg_match('/^\/graphql\/?$/', $uri) );
```